spdx/document.py

```diff
@@ -51,7 +52,7 @@ def from_full_name(cls, full_name):
         the identifier is retrieved from it. Otherwise
         the identifier is the same as the full_name.
         """
-        if full_name in config.LICENSE_MAP.keys():
+        if full_name.encode("utf-8") in config.LICENSE_MAP.keys():
```

Comment:
Based on my overall review comment, this should never be needed here. e.g. if you use unicode throughout, do any needed conversion at the boundaries and not here.